regression/cbmc/integral-trace/test.desc

```diff
@@ -0,0 +1,22 @@
+CORE this_one
+test.c
+--trace --xml-ui
+activate-multi-line-match
+^EXIT=10$
+^SIGNAL=0$
+VERIFICATION FAILED
+<full_lhs>u8</full_lhs>\n\s*<full_lhs_value binary="01100001">97</full_lhs_value>
+<full_lhs>u16</full_lhs>\n\s*<full_lhs_value binary="0{12}1000">8</full_lhs_value>
+<full_lhs>u32</full_lhs>\n\s*<full_lhs_value binary="0{16}0{11}10000">16u</full_lhs_value>
+<full_lhs>u64</full_lhs>\n\s*<full_lhs_value binary="0{32}0{16}0{10}100000">32ul</full_lhs_value>
```

Comment:
FWIW : This is not what would expect to find in a field marked `binary`; I would just expect `0` and `1`.

Reply:
@martin-cs this: `0{32}` is regex notation for repetition, it means 32 0's. I assume this was chosen here to make the structure of the matched expressions somewhat easier to see.

Reply:
Ah; yes.  I missed that it is a regex in the test pattern rather than a literal output.  Thanks.